### DISCUSSION TITLE
Titles not rendering in html when following markdown format on images

### DISCUSSION THREAD

**Discussion by scheric1** [spatie/laravel-markdown#70] [Q&A]
I believe this to be a bug unless I'm doing something wrong.

When I try rendering the following the syntax with laravel-markdown `![test](hello-world.jpg "Hello World!") ` the title "Hello World!" does not render. It actually prevents the image from rendering altogether. Not sure if this is an issue with laravel-markdown or perhaps a dependency that laravel-markdown relies on to render the html from the markdown.

Has anyone else experienced this issue?

**Comment by DjustinK** [comment]
It works fine on my end, it renders: `<img src="hello-world.jpg" alt="test" title="Hello World!">`
